types/index.d.ts

```diff
@@ -572,9 +573,22 @@ declare namespace CbServer {
 
   interface TriggersAsync {
     create(option: Omit<TriggerCreateOptions, "system_key">): Promise<object>;
-    read(query: Query): Promise<object[]>;
-    update(query: Query, changes: object): Promise<object[]>;
-    delete(query: Query): Promise<object>;
+    read(query: AsyncPlatformQuery): Promise<object[]>;
+    update(query: AsyncPlatformQuery, changes: object): Promise<object[]>;
+    delete(query: AsyncPlatformQuery): Promise<object>;
+  }
+
+  interface TimersAsyncCreateOptions extends TimerCreateOptions {
+    name: string;
+    user_id: string;
```

Comment:
According to the [github docs](https://github.com/ClearBlade/native-libraries/blob/master/async.md#timers) user_id and user_type are optional